packages/react-dom/src/__tests__/ReactFunctionComponent-test.js

```diff
@@ -25,6 +26,8 @@ describe('ReactFunctionComponent', () => {
     React = require('react');
     ReactDOM = require('react-dom');
     ReactTestUtils = require('react-dom/test-utils');
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactFeatureFlags.warnAbouDefaultPropsOnFunctionComponents = true;
```

Comment:
s/`warnAbouDefaultPropsOnFunctionComponents`/`warnAboutDefaultPropsOnFunctionComponents` (note the extra t at the end of `About`)